### ISSUE TITLE
Volume adjustments per audio-channel

### ISSUE THREAD

**Issue by 5opr4ni** [CasparCG/Server#652]
Hi

Today you can adjust the volume on a layer in CasparCG like this, `MIXER 1-10 VOLUME 0.5`
To increase the functionality of this it would be awesome to do individual adjustments on available audiochannels on the layer.
I am currently doing a Vision/Audiomixer set up entirely based on CasparCG and this I the only feature missing for making it really simple to do.

With this, the possibility of doing whats called Audio-Mixminus for return audio on OB-Jobs, would be simple.

/o

**Comment by jesperstarkar** [comment]
A theoretical workaround today is to use routing and runtime audio mappings. It will solve simple cases, like mix minus, but it is not a convenient way to work.

**Comment by 5opr4ni** [comment]
Thats what I do at the moment, to get around the issue.

/o

**Comment by jesperstarkar** [comment]
It's a really good feature request. Would be awesome! I'm following up with #653

**Comment by dotarmin** [comment]
Nice request!

**Comment by ronag** [comment]
Do we want this on MIXER level or LOADBG/PLAY level?

**Comment by 5opr4ni** [comment]
Nice! That solves the most situations and I think that the rest can be done with the current implementation. Don’t close it yet, need to think a bit. 
Grate work!

**Comment by jesperstarkar** [comment]
In my opinion: MIXER LEVEL for sure.

It makes sense to have this at the MIXER stage, as there are audio mappings and possible audio configuration mixing going on between producer -> mixer as well as mixer -> consumer.

Effectively this will be a mixer - > mixer audio configuration, implemented the same way as the current configurations, I guess – with runtime AMCP adjustment.

**Comment by 5opr4ni** [comment]
I agree with @jesperstarkar, and I actually had a chat with some radiofolks at musikhjälpen and they said that this possibility could open up a new market for CasparCG.

**Comment by dotarmin** [comment]
I agree with @jesperstarkar, I think it fits best at MIXER LEVEL and have the opportunity to adjust it in runtime using AMCP.

**Comment by TomKaltz** [comment]
Using ffmpeg audio filters on the LOADBG/PLAY level.  Can the level be changed during playback?

**Comment by jesperstarkar** [comment]
The same function should be available over CALL

**Comment by dotarmin** [comment]
Will this also work for decklink input?
